lms/resources/_js_config/__init__.py

```diff
@@ -266,6 +266,9 @@ def enable_dashboard_mode(self):
                         course_assignment_stats=self._to_frontend_template(
                             "api.dashboard.course.assignments.stats"
                         ),
+                        organization_courses=self._to_frontend_template(
```

Comment:
No _stats here yet so no need for that in the name. Should I added it here to avoid extra work in the future?

Reply:
Perhaps we should not have added `_stats` to any of them, to leave more room to decide the exact information returned for each one of them.

Let's leave it like this for now, and we can see in future if we need to make adjustments.